crates/rover-client/src/error.rs

```diff
@@ -20,12 +20,34 @@ pub enum RoverClientError {
     InvalidJSON(#[from] serde_json::Error),
 
     /// Encountered an error handling the received response.
-    #[error("encountered an error handling the response: {msg}")]
-    HandleResponse {
+    #[error("{msg}")]
+    AdhocError {
```

Comment:
How about calling this `RoverInternalError`? We can then say something along the lines of, if you encounter this, please submit an issue to our issue tracker.
```suggestion
    RoverInternalError {
```

Reply:
Ohhh I see, this is used for errors from Studio, essentially. And we provide as detailed as possible description. You can ignore my point about 'submitting an issue to our issue tracker'.